vault/logical_system.go

```diff
@@ -607,11 +609,28 @@ func (b *SystemBackend) handleRekeyRetrieve(
 		return logical.ErrorResponse("no backed-up keys found"), nil
 	}
 
+	keysB64 := map[string][]string{}
```

Comment:
At this time I'd prefer not to change the structure of the backup. Not because we can't but because it's late in the cycle and I don't want to do anything that might break. So I'm populating this at fetch time.